src/vmci_connecter.cpp

```diff
@@ -100,15 +100,15 @@ void zmq::vmci_connecter_t::process_term (int linger_)
     own_t::process_term (linger_);
 }
 
-void zmq::vmci_connecter_t::in_event ()
+void zmq::vmci_connecter_t::in_event (i_poll_events::handle_t handle_)
 {
     //  We are not polling for incoming data, so we are actually called
     //  because of error here. However, we can get error on out event as well
     //  on some platforms, so we'll simply handle both events in the same way.
-    out_event ();
+    out_event (i_poll_events::handle_t handle_);
```

Comment:
This doesn't look as this was intended. Does this event compile?

Should probably be 
```
out_event (handle_);
```